.github/workflows/uploadFirrtlReleaseArtifacts.yml

```diff
@@ -41,25 +44,36 @@ jobs:
             assert: OFF
             shared: OFF
             stats: ON
-            cmake-args: ''
-        runner: [ubuntu-20.04, ubuntu-18.04, macos-11]
+        runner: [windows-2019, ubuntu-20.04, macos-11]
         include:
           - runner: ubuntu-20.04
             os: linux
-            tar: tar
+            arch: x64
+            tar: tar czf
+            archive: tar.gz
+            sha256: shasum -a 256
+            cont: "\\"
+            setup: ""
             # Default clang (11) is broken, see LLVM issue 59622.
-            cc: clang-12
-            cxx: clang++-12
-          - runner: ubuntu-18.04
-            os: linux
-            tar: tar
-            cc: clang
-            cxx: clang++
+            cmake-args: '-DCMAKE_C_COMPILER=clang-12 -DCMAKE_CXX_COMPILER=clang++-12'
```

Comment:
I moved the compiler override here because I couldn't figure out what the name of the compiler was for Windows 😂. Open to moving it back but it is also nice having MacOS and Windows just use the default rather than having to also specify it for them.